driver/src/main/java/org/neo4j/driver/v1/Entry.java

```diff
@@ -21,11 +21,12 @@
 /**
  * Immutable pair of a key and a value
  *
- * @see PropertyMapAccessor
+ * @see MapAccessor
+ * @see RecordAccessor
  * @param <V> the Java type of the contained value
  */
 @Immutable
-public interface Property<V>
+public interface Entry<V>
```

Comment:
Wikipedia agrees with that idea: https://en.wikipedia.org/wiki/Associative_array